LogMonitor/src/LogMonitor/Main.cpp

```diff
@@ -230,6 +230,7 @@ int __cdecl wmain(int argc, WCHAR *argv[])
 {
     std::wstring cmdline;
     PWCHAR configFileName = (PWCHAR)DEFAULT_CONFIG_FILENAME;
+    int exitcode = 0;

```

Comment:
Probably not a big issue but we can have this as ExitCode in future @iankingori

Reply:
Any reason for the capital letters or should this be `exit_code`?

Reply:
I'll have to confirm with the C++ conventions for this, but I guess it comes down to preference at this point